spec/lib/sentry/processor/pii_sanitizer_processor_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'sentry/processor/email_sanitizer'
```

Comment:
not needed

Reply:
Whoops. Pulled that import accidentally from the other one. Thanks.